modules/installation-and-upgrade/pages/install-proxy-vm.adoc

```diff
@@ -0,0 +1,112 @@
+[[install-proxy-vm]]
+= Install {productname} Proxy in a Virtual Machine Environment Using {productname} Proxy Image
+
+
+
+== Virtual Machine Manager (virt-manager) Settings
+
+This chapter provides the required Kernel Virtual Machine (KVM) settings for {productname} Proxy.
+KVM combined with Virtual Machine Manager (_virt-manager_) will be used as a sandbox for this installation.
+
+
+
+You will find the VM images for {productname} {productnumber} Proxy in various formats.
+It includes the underlying OS bits ({sles}_{sles-version} {sp-base}) and the {productname} Proxy software current at the time of build.
+Download the appropriate {productname} Proxy image for your environment from https://download.suse.com/.
+
+[NOTE]
+====
+This table specifies the minimum requirements.
+These are suitable for a quick test installation, such as a server with one client.
+If you want to use a production environment, review the requirements listed in xref:installation-and-upgrade:hardware-requirements.adoc[].
+====
+
+
+[cols="1,1", options="header"]
+|===
+2+<| Virtual machine settings overview
+| Installation Method | Import Existing Disk Image
+| OS:                 | SUSE Linux Enterprise 15 SP4
+| Memory:             | 8 GB
+| CPU's:              | 2
+| Virtual Disks:      |
+| VirtIO Disk 1       | SUSE-Manager-Proxy.x86_64-4.3.13-KVM.qcow2
+| VirtIO Disk 2       | 101 GB for [path]``/srv/``
+| VirtIO Disk 3       | 100 GB for [path]``/srv/cache/``
+| Name:               | suse-manager-proxy-setup
+| Network             | Bridge _br0_
+|===
+
+[NOTE]
+====
+For more information on {sle} Virtualization, see https://documentation.suse.com/sles/15-SP4/html/SLES-all/book-virtualization.html.
+====
+
+
+
+== {productname} Proxy Virtual Machine Settings
```

Comment:
I find this title a bit confusing. Chapter on is about Virtual Machine Manager Settings, but the settings below are somehow also for those.

Reply:
I hope I found a better title now.